python/lsst/ts/rubintv/s3client.py

```diff
@@ -39,7 +49,7 @@ def list_objects(self, prefix: str) -> list[dict[str, str]]:
         return objects
 
     def _get_object(self, key: str) -> dict | None:
-        logger = structlog.get_logger(__name__)
+        logger = structlog.get_logger("rubintv")
```

Comment:
Maybe you can define a logger at the beginning of this file as you have in other files in the project.